classes/requests/post/class-kp-unavailable-features.php

```diff
@@ -71,15 +71,15 @@ protected function get_body() {
 				'platform_data'      => array(
 					'platform_name'        => 'woocommerce',
 					'platform_version'     => WC()->version,
-					'platform_plugin_name' => 'woocommerce_klarna_payments',
+					'platform_plugin_name' => 'woocommerce_klarna_checkout_plugin',
```

Comment:
I checked with Klarna today, and this value should probably be `woocommerce_klarna_plugin` instead.